x-pack/plugins/apm/server/index.ts

```diff
@@ -21,7 +21,7 @@ export const config = {
     serviceMapFingerprintBucketSize: schema.number({ defaultValue: 100 }),
     serviceMapTraceIdBucketSize: schema.number({ defaultValue: 65 }),
     serviceMapFingerprintGlobalBucketSize: schema.number({
-      defaultValue: 1000,
+      defaultValue: 500,
```

Comment:
As we talked about, this seems like a fix that's tailor made for our setup. Would be better to find something that didn't rely on an arbitrary number (if that's possible).